src/main/java/com/github/devotedmc/hiddenore/ToolConfig.java

```diff
@@ -62,16 +61,15 @@
 	private double maxAmountModifier;
 	private double dropChanceModifier;
 	
-	private static Map<String, ToolConfig> tools = new HashMap<String, ToolConfig>();
-	private static List<String> toolList = new LinkedList<String>();
+	private static Map<String, ToolConfig> tools = new HashMap<>();
+	private static List<String> toolList = new LinkedList<>();
 	
 	protected ToolConfig(ItemStack template, boolean ignoreAmount, boolean ignoreDurability,
 			boolean ignoreEnchants, boolean ignoreOtherEnchants, boolean ignoreEnchantsLvl, 
 			boolean ignoreLore, boolean ignoreName,
 			Double dropChanceModifier, Double minAmountModifier, Double maxAmountModifier) {
 		this.template = template;
 		this.ignoreAmount = ignoreAmount;
-		this.ignoreDurability = ignoreDurability;
```

Comment:
Can't remember 100% but I seem to recall this durability had implications for actual durability -- e.g. to allow "last hit" effects. So removing it seems... Not Ok.

Reply:
Oh yeah right, I was in the mode of "durability = gone", my bad.